homeassistant/components/tuya/switch.py

```diff
@@ -634,6 +654,10 @@
     # Humidifier
     # https://developer.tuya.com/en/docs/iot/categoryjsq?id=Kaiuz1smr440b
     "jsq": (
+        SwitchEntityDescription(
+            key=DPCode.SWITCH,
+            name="Power",
+        ),
```

Comment:
Support for toggling/switching for humidifiers should be provided by the `humidifier` entity, not the `switch` domain.